### ISSUE TITLE
[CoE Starter Kit - QUESTION] Environment refresh

### ISSUE THREAD

**Issue by SOMSGH** [microsoft/coe-starter-kit#6820]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hi,

I've exempt one specific environment from inventory , but after 24 hours the COE environment table still keeping the data of this environment, I would like to know which flow working on removing environments and when will the environment exclusion from inventory be updated? Thank you!

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

September 2023

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
This is cleaned up by runs of _**CLEANUP - Admin | Sync Template v3 (Delete Bad Data)**_ which should run daily. 

If that does not fix it up please go to the Admin Command Center > CoE Flows > Inventory. Validate that all flows are turned on and that there are no unmanaged layers as shown here. 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/598b5024-2e1a-4390-a6b2-49644368c246)

**Comment by SOMSGH** [comment]
Hi @Jenefer-Monroe ,

I've checked all inventory flows are on and no unmanaged layer.
I reproduce quickly by exclude one environment from inventory at admin view app> manually run CLEANUP - Admin | Sync Template v3 (Delete Bad Data) > refresh the BI dashboard, then found fact is the the flows and apps from this excluded environment (named 'Host environment')are removed from Admin view app , the inventory excluded environment will be shown at admin view > environment excused from inventory. (This inventory excluded environment record stays at COE admin_environment table.)

![image](https://github.com/microsoft/coe-starter-kit/assets/131751950/a33a6f30-6830-46ec-87d3-75ddbf7f6955)

However, at BI side you will also find it still under environment and environment capacity pages
![image](https://github.com/microsoft/coe-starter-kit/assets/131751950/86c2af86-b45e-4e95-b19c-4345deaeeb92)
![image](https://github.com/microsoft/coe-starter-kit/assets/131751950/caa120c5-9406-415b-87cf-0f27c562f9d2)


So I'm wondering whether why the inventory excluded environment is showing at BI dashboard?

**Comment by Jenefer-Monroe** [comment]
yep looks like we just missed this feature for the PBIs. posting for Nov release.

**Comment by Jenefer-Monroe** [comment]
Please turn the delete bad data flow off and back on, perhaps the old env var is cached there. 
Rerun it and let me know if this is resolved

**Comment by SOMSGH** [comment]
thanks, it works now
